wled00/FX_fcn.cpp

```diff
@@ -77,8 +77,8 @@ uint16_t Segment::_usedSegmentData = 0U; // amount of RAM all segments use for t
 uint16_t Segment::maxWidth = DEFAULT_LED_COUNT;
 uint16_t Segment::maxHeight = 1;
 
-CRGBPalette16 Segment::_randomPalette = generateRandomPalette(&_randomPalette);
-CRGBPalette16 Segment::_newRandomPalette = generateRandomPalette(&_randomPalette);
+CRGBPalette16 Segment::_randomPalette = generateRandomPalette(_randomPalette);
```

Comment:
Recursive initialisation is unacceptable IMO.

Reply:
can use any palette available. is there a global one that can be used?

Reply:
I commented this before the single color one :) still need to try that but should be fine